pkg/volume/portworx/portworx_util.go

```diff
@@ -181,13 +186,24 @@ func createDriverClient(hostname string) (*osdclient.Client, error) {
 	}
 }
 
-func (util *PortworxVolumeUtil) getPortworxDriver(volumeHost volume.VolumeHost) (volumeapi.VolumeDriver, error) {
+func (util *PortworxVolumeUtil) getPortworxDriver(volumeHost volume.VolumeHost, localOnly bool) (volumeapi.VolumeDriver, error) {
```

Comment:
Please see if the update comment gives more context. I have given a high-level context on why create and delete calls can use localOnly as false while other's cannot.

Currently, we don't have external docs describing our internal algorithms (most of them are proprietary.)